test/upgrade/mzcompose.py

```diff
@@ -83,11 +83,21 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         priors = [f"v{v}" for v in all_versions if v < version]
         test_upgrade_from_version(c, f"v{version}", priors, filter=args.filter)
 
-    test_upgrade_from_version(c, "current_source", priors=["*"], filter=args.filter)
+    test_upgrade_from_version(
+        c,
+        "current_source",
+        priors=["*"],
+        filter=args.filter,
+        check_from_accept_skip_ifs=True,
```

Comment:
You may wish to mention this in `test/upgrade/README.md` as it is likely that future PRs will also need similar functionality.